assets/dir-index-html/src/dir-index.html

```diff
@@ -52,10 +52,13 @@
           {{ .Hash }}
         </div>
         {{ end }}
+        <p style="margin-bottom: 0">
+          <a href="?format=json">Preview as JSON.</a> Download as: <a href="?format=car">CAR</a>, <a href="?format=tar">TAR</a>, <a href="?format=dag-json">DAG-JSON</a>, <a href="?format=dag-cbor">DAG-CBOR</a>.
+        </p>
       </div>
       {{ if .Size }}
       <div class="no-linebreak flex-shrink-1 ml-auto">
-        <strong>&nbsp;{{ .Size }}</strong>
+        <strong title="Cumulative Object Size">&nbsp;{{ .Size }}</strong>
```

Comment:
Bit more clear what it means (total size of entire DAG)

```suggestion
        <strong title="Cumulative size of IPFS DAG (data + metadata)">&nbsp;{{ .Size }}</strong>
```